.github/workflows/build-check.yaml

```diff
@@ -82,3 +82,22 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - run: make gen-kotlin
+
+  breaking-change:
+    needs: docker-pull
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    # This is needed because checkout action does a shallow clone
+    - name: Fetch main branch for comparison
+      run: |
+        git remote set-branches origin '*'
+        git fetch --depth=1 origin main
+        git branch main origin/main
+    - name: Run make breaking-change with json output to annotate PR
```

Comment:
I agree that this is very helpful, thank you!

It would be nice if you could add a breaking change in a new (dummy) PR based off this branch to illustrate how this would work.

Reply:
https://github.com/aabmass/opentelemetry-proto/pull/5/files#diff-62ba46dcb97aa81748b276b6eb42ae8e6700cd4b467515542539435eda06d01f

Reply:
Looks good, thank you!